uzupelnianka.txt

```diff
@@ -1,7 +1,7 @@
-Wyjątki dzielimy na … które trzeba obsłużyć i … które można, ale nie trzeba obsłużyć.
+Wyjątki dzielimy na kontrolowane, które trzeba obsłużyć i niekontrolowane które można, ale nie trzeba obsłużyć.
 
-Wyjątki … dziedziczą po klasie RuntimeException
+Wyjątki niekontrolowane dziedziczą po klasie RuntimeException
 
-Wyjątki rzucamy korzystając z instrukcji …, jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa …
+Wyjątki rzucamy korzystając z instrukcji throw new JakiśTamWyjątek, jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa throws.
```

Comment:
> throw new JakiśTamWyjątek

Nie do końca. Samo `throw` oznacza rzucanie wyjątków. Reszta to jego tworzenie, tak samo jak każdego innego obiektu.

```
Exception exception = new RuntimeException();
throw exception;
```